src/js/components/Menu.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Drawer from 'material-ui/Drawer';
+import { Link } from 'react-router';
+import MenuItem from 'material-ui/MenuItem';
+
+const Menu = (props) => (
+  <Drawer
+    docked={false}
+    open={props.open}
+    onRequestChange={(open) => props.drawerChange(open)}
+  >
+    <Link to={'/'}>
+      <MenuItem onTouchTap={props.close} primaryText="Search" />
```

Comment:
Good to know, I can change that, sure. Most things are here because they're suggestive - related to the specs page in the wiki.

Some things we should settle on:
* where does the Search bar go?
* what happens to the search bar when there's content we need to load (e.g. About, Legend, API).
* if it sticks around, what happens when we search again?

Could be some UX concerns about how to handle "static" pages (e.g. About, Legend, API) when displayed adjacent to the search bar. For instance, what happens when you search, again? Does the static page disappear?